cypress/support/fragments/invoices/settingsInvoices.js

```diff
@@ -0,0 +1,62 @@
+import { Button, TextField, NavListItem, MultiColumnListHeader, EditableListRow, MultiColumnListCell } from '../../../../interactors';
+import InteractorsTools from '../../utils/interactorsTools';
+
+const settings = {
+  approvals: 'Approvals',
+  adjustments: 'Adjustments',
+  batchGroups: 'Batch groups',
+  batchGroupConfiguration: 'Batch group configuration',
+  voucherNumber: 'Voucher number'
+};
+
+export default {
+  settings: {
+    approvals: 'Approvals',
+    adjustments: 'Adjustments',
+    batchGroups: 'Batch groups',
+    batchGroupConfiguration: 'Batch group configuration',
+    voucherNumber: 'Voucher number'
+  },
+
+  selectSetting: (setting) => {
+    cy.do(NavListItem(setting).click());
+    cy.expect(MultiColumnListHeader({ id: 'list-column-name' }).exists());
+  },
+
+  createNewBatchGroup: (batchGroup) => {
+    cy.expect(Button({ id: 'clickable-add-batch-groups' }).exists());
```

Comment:
do you really  need this waiter? if yes - move it into waitLoading() and use it after step with fragment opening

Reply:
Updated